### PR TITLE
changed source to sources https://issues.redhat.com/browse/ACM-6811

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira issue: https://issues.redhat.com/browse/ACM-6811

### CODE REVIEW

**Review by dockerymick — CHANGES_REQUESTED** [review]
This looks good. I ran the link check tool and noticed a broken link

Screenshot 2023-08-30 at 2 20 25 PM

Will you make the following update (remove double asterisk in all instances to make sure the format is consistent. Double asterisks can affect the doc): 

Change: **Note:** For more details on the prerequisites for deploying `ApplicationSet`, see link:../gitops/gitops_config.adoc#gitops-register[Registering managed clusters to GitOps].


TO 

*Note:* For more details on the prerequisites for deploying `ApplicationSet`, see link:../gitops/gitops_registering.adoc#gitops-register[Registering managed clusters to GitOps].

**Comment by dockerymick** [comment]
@jc-berger will you remove the documentation label? We don't use labels for PRs

File: applications/app_model.adoc [hunk 1]
```diff
@@ -49,7 +49,7 @@ If you have applications that require Kubernetes resources or Helm charts from c
 
 Channels use a namespace within the hub cluster and point to a physical place where resources are stored for deployment. Clusters can subscribe to channels for identifying the resources to deploy to each cluster.
 
-**Notes:** It is best practice to create each channel in a unique namespace. However, a Git channel can share a namespace with another type of channel, including Git, Helm, and Object storage.
+*Notes:** It is best practice to create each channel in a unique namespace. However, a Git channel can share a namespace with another type of channel, including Git, Helm, and Object storage.
```
Comment (dockerymick) [inline applications/app_model.adoc]:
```suggestion
*Notes:* It is best practice to create each channel in a unique namespace. However, a Git channel can share a namespace with another type of channel, including Git, Helm, and Object storage.
```

File: applications/app_model.adoc [hunk 2]
```diff
@@ -66,7 +66,7 @@ Resources within a channel can be accessed by only the clusters that subscribe t
 
 `ApplicationSet` is a sub-project of Argo CD that is supported by the GitOps Operator. `ApplicationSet` adds multicluster support for Argo CD applications. You can create an application set from the {product-title-short} console.
 
-**Note:** For more details on the prerequisites for deploying `ApplicationSet`, see link:../gitops/gitops_config.adoc#gitops-register[Registering managed clusters to GitOps].
+*Note:** For more details on the prerequisites for deploying `ApplicationSet`, see link:../gitops/gitops_registering.adoc#gitops-register[Registering managed clusters to GitOps].
```
Comment (dockerymick) [inline applications/app_model.adoc]:
```suggestion
*Note:* For more details on the prerequisites for deploying `ApplicationSet`, see link:../gitops/gitops_registering.adoc#gitops-register[Registering managed clusters to GitOps].
```

**Review by dockerymick — APPROVED** [review]
Added suggestions and checked links. Remove the extra asterisks. Otherwise this looks good!

File: gitops/gitops_push_pull.adoc [hunk 3]
```diff
@@ -145,9 +145,11 @@ spec:
         namespace: guestbook
         server: https://kubernetes.default.svc
       project: default
-      source:
-        path: guestbook
-        repoURL: https://github.com/argoproj/argocd-example-apps.git
+      sources: [
+      { repoURL: https://github.com/argoproj/argocd-example-apps.git
+        targetRevision: main
+        path: guestbook }
```
Comment (chenz4027) [inline gitops/gitops_push_pull.adoc]: Would it be possible to move } to the next line?

File: applications/app_model.adoc [hunk 4]
```diff
@@ -105,7 +105,7 @@ spec:
     spec:
       project: default
       sources: [
-      {  repoURL: https://github.com/sampleapp/apprepo.git
+      { repoURL: https://github.com/sampleapp/apprepo.git
         targetRevision: main
         path: sample-application }
```
Comment (chenz4027) [inline applications/app_model.adoc]: same as below

**Review by chenz4027 — APPROVED** [review]
/lgtm

**Pull request merged**